scripts/safari.js

```diff
@@ -3853,6 +3872,22 @@ function Safari() {
             }
             this.addToMonthlyLeaderboards(player.id, "pokesCaught", 1);
 
+            if (ball === "myth") {
+                player.records.catchMyth += 1;
+            }
+            if (ball === "heavy") {
+                player.records.catchHeavy += 1;
+            }
+            if (ball === "quick") {
+                player.records.catchQuick += 1;
+            }
+            if (ball === "mono") {
+                player.records.catchMono += 1;
+            }
+            if (ball === "clone") {
+                player.records.catchClone += 1;
+            }
+
             var clonedAmount = 0;
             if (ball === "clone" || crystalEffect.effect === "clone") {
                 var costumed = player.costume === "scientist";
```

Comment:
you can place `player.records.catchClone += 1;` under here instead of doing another check above